src/Parser/HolyPriest/Modules/PriestCore/Serendipity.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage, formatNumber } from 'common/format';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+// dependencies
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SerenityReduction from './SerendipityReduction/SerenityReduction';
+import SanctifyReduction from './SerendipityReduction/SanctifyReduction';
+
+// This module might contain too much information but to rewrite some of the output
+// in this file would entail a lot of redundant copying (aka violating the "DRY"
+// coding mindset). Let me @enragednuke if you think I should move stuff out.
+
+// Currently contains:
+// - "Wasted Serendipity" statistic
+// - "Fully missed holy word casts (via Serendipity)" suggestion
+// - "Holy Priest T20 2P Bonus" item
+class Serendipity extends Module {
+  static dependencies = {
+    combatants: Combatants,
+    serenity: SerenityReduction,
+    sanctify: SanctifyReduction,
+  }
+
+  statistic() {
+    const formatted = {
```

Comment:
Not sure that you need to make this 'formatted' structure. Most of these calculations are pretty short, why not do them inline? Or if you want the tooltip cleaner just make a bunch of const vals? Basically it looks like you made this a nested object for no good reason.

Reply:
Just for a cleaner look than a bunch of camel cased variables, no other reason.